### DISCUSSION TITLE
Restrict Button in Object Editor to Display Only for Parent Products, Not Variants(Child Product)

### DISCUSSION THREAD

**Discussion by Muhammed-Rishan** [pimcore/pimcore#17663] [Q&A / Support]
I have created a button in the object editor that should only be visible for the parent product. However, it is currently also displaying for variant (child) products. How can I restrict the button to ensure it only appears for the parent product and not its variants?

**Comment by kingjia90** [comment]
Could you please paste a snippet of the code that hints where is this button located?  
The type should be easily extracted by the object like
https://github.com/pimcore/admin-ui-classic-bundle/blob/5ea0e1ddb2a149937b3cfa05a7b811bd603bdad2/public/js/pimcore/element/helpers/gridColumnConfig.js#L409
